guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,21 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Clevis/Tang during host provisioning.
+You must associate the partition table with your operating system entry.
+Then, you assign it to your host group or select it manually during provisioning.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+
+.Next steps
+* Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
+* Add host parameters to your host group to configure disk encryption:
+** `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as value (default: `linux`)
```

Comment:
"... and your cleartext passphrase for the LUKS container ... . If not specified, the passphrase is set to `linux`."

Reply:
Applied.